src/Atmos/Model/source.jl

```diff
@@ -38,33 +38,40 @@ struct GeostrophicForcing{FT} <: Source
   u_geostrophic::FT
   v_geostrophic::FT
 end
-function atmos_source!(s::GeostrophicForcing, m::AtmosModel, source::Vars, state::Vars, aux::Vars, t::Real)
-  u = state.ρu / state.ρ
+function atmos_source!(s::GeostrophicForcing, atmos::AtmosModel, source::Vars, state::Vars, aux::Vars, t::Real)
+  ρ = state.ρ
+  u = state.ρu/ρ
   u_geo = SVector(s.u_geostrophic, s.v_geostrophic, 0)
-  source.ρu -= state.ρ * s.f_coriolis * (u - u_geo)
+  ẑ = vertical_unit_vector(atmos.orientation, aux)
+  fkvector = s.f_coriolis * ẑ
+  source.ρu -= ρ * fkvector × (u .- u_geo)
```

Comment:
Unless I did the algebra wrong, I believe this should have a `+` sign and not a `-` sign.

The momentum equation should be:

DU/Dt = \rho f (v_g - v)
DV/Dt = \rho f (u   - ug)

but, with the - as written above, the RHS changes.

Reply:
Here are the momentum equations: 
<img width="275" alt="Screen Shot 2020-01-13 at 5 12 27 PM" src="https://user-images.githubusercontent.com/15185316/72305228-f4636300-3627-11ea-8039-11c9f5286b85.png">
(I.e., opposite sign of what you have above). The Coriolis term above looks correct to me.

Reply:
Or, another way of looking at it is that the Coriolis force on the RHS is `-2\Omega \times \rho u`, which is what we have (minus the subtraction of the background state).